x-pack/legacy/plugins/transform/public/app/sections/create_transform/components/step_create/step_create_form.tsx

```diff
@@ -127,25 +131,32 @@ export const StepCreateForm: FC<Props> = React.memo(
     }
 
     async function startTransform() {
-      setStarted(true);
+      setLoading(true);
 
       try {
-        await api.startTransforms([{ id: transformId }]);
-        toastNotifications.addSuccess(
-          i18n.translate('xpack.transform.stepCreateForm.startTransformSuccessMessage', {
-            defaultMessage: 'Request to start transform {transformId} acknowledged.',
-            values: { transformId },
-          })
-        );
+        const resp = await api.startTransforms([{ id: transformId }]);
+        if (typeof resp === 'object' && resp !== null && resp[transformId]?.success === true) {
+          toastNotifications.addSuccess(
+            i18n.translate('xpack.transform.stepCreateForm.startTransformSuccessMessage', {
+              defaultMessage: 'Request to start transform {transformId} acknowledged.',
+              values: { transformId },
+            })
+          );
+          setStarted(true);
+          setLoading(false);
+        } else {
+          throw new Error();
```

Comment:
this error contains no text so it looks like the toast body below will be empty?

Reply:
Fixed in f080f31.